index.js

```diff
@@ -0,0 +1,70 @@
+const books = [
+  {
+    title: 'first',
+    author: 'name',
+  },
+  {
+    title: 'second',
+    author: 'name',
+  },
+  {
+    title: 'third',
+    author: 'name',
+  },
+];
+
+// form declaration
+
+const form = document.querySelector('#form');
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+const formButton = document.querySelector('#submit-button');
+const bookList = document.querySelector('ul');
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  form.submit();
+});
+
+// load each book form the local array
+
+const removeBook = (bookItem) => {
+  const removeBtn = bookItem.querySelector('.remove');
+  removeBtn.addEventListener('click', () => {
+    books.forEach((book, index) => {
+      if (book.title === bookItem.textContent.split('by')[0].trim()) {
+        books.splice(index, 1);
+        localStorage.setItem('books', JSON.stringify(books));
+      }
+    });
+    bookItem.remove();
+  });
+};
+
+window.onload = () => {
+  books.forEach((book) => {
+    const bookItem = document.createElement('li');
+    bookItem.innerHTML = `${book.title} by ${book.author} <button class="remove">Remove</button>`;
+    bookList.appendChild(bookItem);
+    removeBook(bookItem);
+  });
+};
+
```

Comment:
- [ ] The same issue highlighted by the previous reviewer is still there. When books are added and page is refreshed the books are removed from the page. What we have is your own defined books at the top of this file. Kindly refactor your code so that when the page is reloaded it loads the books from the local storage and presents them on the page, instead of your own book.